src/lib/builders.ts

```diff
@@ -94,17 +82,6 @@ export class ItemBuilder {
         this.urls = { primaryUrl: "", hrefs: [] };
     }
 
-    /**
-     * Sets the parent Vault ID for the Item being constructed.
-     *
-     * @param {string} vaultId
-     * @returns {ItemBuilder}
-     */
-    public setVault(vaultId: string): ItemBuilder {
```

Comment:
Realized if we may break customer's integrations if we remove this. We could keep backward compatibility for the next release if you change the method body to do a no-op:
```
return this;
```

Could you also add a deprecation notice to the doc block? https://jsdoc.app/tags-deprecated.html 

We can fully remove it in the next version :)

Reply:
Makes sense to first deprecate it and then fully remove it.
FIxed.